### PR TITLE
Add all current stable releases of Ruby to matrix

### PR DESCRIPTION
**Pull request by freshtonic**
See: https://www.ruby-lang.org/en/downloads

### CODE REVIEW

File: lib/active_stash/collection_proxy.rb [hunk 1]
```diff
@@ -64,10 +64,14 @@ def consistency_check!(collection)
       end
 
       def client
-        @client ||= CipherStash::Client.new(
-          logger: ActiveStash::Logger.instance,
-          **ActiveStash.config.to_client_opts
-        )
+        # @client ||= CipherStash::Client.new(
```
Comment (mpalmer) [inline lib/active_stash/collection_proxy.rb:67]: :raised_eyebrow: debug leftovers?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -7,6 +7,7 @@ jobs:
       fail-fast: false
       matrix:
         rails-version: [6, 7]
+        ruby-version: [2.7.6, 3.0.4, 3.1.2]
```
Comment (mpalmer) [inline .github/workflows/build.yml:10]:
```suggestion
        ruby-version: [2.7, 3.0, 3.1]
```

While this gives us fine-grained control over the exact Ruby versions used in a given test run, I've found in the past that it's extremely tedious to be constantly updating the workflow file whenever new Ruby versions come out, to very little benefit.  The `ruby/setup-ruby` action accepts the MAJ.MIN form quite happily, according to the docs.

**Review by mpalmer — CHANGES_REQUESTED** [review]
Unrelated-looking commented out code needs fixing, or explaining, or something, IMO.

**Comment by freshtonic** [comment]
@mpalmer this is supposed to be a draft PR. Not sure what happened there.

**Comment by mpalmer** [comment]
Not sure why you closed the PR, it's a good idea.

**Pull request closed without merging**